docs/instalation/instalation.md

```diff
@@ -3,24 +3,49 @@ id: instalation
 title: Instalation
 ---
 
-Here's how to install spotify-web-sdk library in your project:
-
+## How to install
 
+Here's how to install spotify-web-sdk library and use in your project:
 
 ### Using Yarn:
-
 ```sh
 yarn add spotify-web-sdk
 ```
-
 ### Using NPM:
-
 ```sh
 npm install spotify-web-sdk
 ```
-
 ### Using CDN:
-
 ```html
 <script src="https://unpkg.com/spotify-web-sdk/build/dist/spotify-web-sdk.min.js"></script>
 ```
+
+## Usage
+
+To get started using spotify-web-sdk, you'll need a Spotify authorization token. Check Spotify's Web API's [Authorization Guide](https://developer.spotify.com/documentation/general/guides/authorization-guide/) to get your token and then get started. Note that there are some specifics authorization scopes, they can be used in different situations, and will deppend of the requests you'll be running.  
+Then, to make full use of the functions spotify-web-sdk makes avaliable, first you need to run the init function. To do that, you will need to use your authorization token, to "log in". After this, our library will use your token to make request of your choice.  
+Here's an example of the usage, using the init() and getTrack() functions of the library:
+
+```javascript
+import * as spotify from 'spotify-web-sdk';
+spotify.init({ token: 'YOUR SPOTIFY TOKEN HERE!' }); // You should only need to run this once.
+spotify.getTrack('3LOpHuEpjkL4T1Zcjhko8w'); // Or any other track id.
+```
+
```

Comment:
I think is a good idea to explain that if you want to know more about the other functionalities of the SDK, you can check the next sections...